### DISCUSSION TITLE
Welcome to gitopener.vercel.app Discussions!

### DISCUSSION THREAD

**Discussion by Aadarsh805** [pbclife/gitopener.vercel.app#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aadarsh805** [comment]
@sboy99 The `/contributors` route is giving `404 not found` why is that,
Main branch doesn't have the data ?

**Reply by sboy99** [reply]
> @sboy99 The `/contributors` route is giving `404 not found` why is that, Main branch doesn't have the data ?

Are you running it on your local machine?

**Reply by Aadarsh805** [reply]
yes sir

**Reply by sboy99** [reply]
Under the hood it's trying to connect with the original database that have contributors data, as your local database has no contributors data this not found page is thrown. To fix that you can create a contributors folder in the root of the directory and add some contribution, make sure the file name matches your github username. And ensure to put frontmatter as following
```
---
author: Aadarsh Thakur
github_username: Aadarsh805
occupation: <your occupation>
---
<write your markdown content>
```
But first you need to add environment variables as mentioned..
```.env
MONGO_URI:
GITHUB_TOKEN:
```

**Reply by Aadarsh805** [reply]
Alright got it
